microsoft-edge/dualengine/get-started.md

```diff
@@ -0,0 +1,48 @@
+---
+title: Getting started with the DualEngine API
+description: Get started using the Microsoft DualEngine API to embed and control an Internet Explorer instance within your app, to provide a browsing experience that's equivalent to a normal Internet Explorer tab, providing increased compatibility with legacy websites.
+author: MSEdgeTeam
+ms.author: msedgedevrel
+ms.topic: conceptual
+ms.service: microsoft-edge
+ms.subservice: windows-integration
+ms.localizationpriority: high
+ms.date: 11/17/2023
+---
+# Getting started with the DualEngine API
+
+This article walks you through the steps to start using the DualEngine API.
+
+
+<!-- ====================================================================== -->
+## Step 1: Get access to the DualEngine Limited Access Feature
+
+Access to the DualEngine API is controlled via a Limited Access Feature (LAF).  To gain access to the feature, reach out to dualengineapiaccess@microsoft.com.
+
+The DualEngine LAF is handled somewhat atypically; for details, see [Unlock the Limited Access Feature](concepts/adapter-dll.md#unlock-the-limited-access-feature) in _Creating a DualEngine adapter plugin DLL_.
+
+
+<!-- ====================================================================== -->
+## Step 2: Download the DualEngine API header
+
+The DualEngine API is a COM interface that's only officially supported via C++, and is not part of the normal Windows SDK.  Therefore, you must include the following header in your adapter DLL project: [DualEngine API Header Download](https://download.microsoft.com/download/c/5/0/c5035487-bd78-4fd0-9cc4-e1c5a3b654b7/dualengine.h).

```

Comment:
this text doesn't show the filename, dualengine.h
Rendered section:
https://review.learn.microsoft.com/en-us/microsoft-edge/dualengine/get-started?branch=pr-en-us-2859#step-2-download-the-dualengine-api-header

Reply:
```
Therefore, you must download and include the DualEngine API header `dualengine.h` 
in your adapter DLL project: 
[Download dualengine.h](https://download.microsoft.com/download/c/5/0/c5035487-bd78-4fd0-9cc4-e1c5a3b654b7/dualengine.h).
```